staging/src/k8s.io/apiserver/pkg/storage/value/encrypt/envelope/v1beta1/service.proto

```diff
@@ -1,3 +1,19 @@
+/*
+Copyright 2018 The Kubernetes Authors.
```

Comment:
This file was added in 2018: https://github.com/kubernetes/kubernetes/pull/55684.